### PR TITLE
Add PSBP (old name: PDBP) to community build

### PR DESCRIPTION
**Pull request by abgruszecki**
@psbp-library thanks for reaching out to (re)include your project in the community build, here's the PR.

### CODE REVIEW

**Comment by abgruszecki** [comment]
@OlivierBlanvillain for some reason, cloning the new repo fails with:
```
Cloning into '/__w/dotty/dotty/community-build/community-projects/psbp'...
Host key verification failed.
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
fatal: clone of 'git@github.com:PSBP-Library/psbp-library.github.io.git' into submodule path '/__w/dotty/dotty/community-build/community-projects/psbp' failed
Failed to clone 'community-build/community-projects/psbp' a second time, aborting
```

This is despite the repo being public. I can clone it locally no problem. Do you have an idea what could be going on?

File: .gitmodules [hunk 1]
```diff
@@ -193,3 +190,6 @@
 [submodule "community-build/community-projects/cask"]
 	path = community-build/community-projects/cask
 	url = https://github.com/dotty-staging/cask.git
+[submodule "community-build/community-projects/psbp"]
+	path = community-build/community-projects/psbp
+	url = git@github.com:PSBP-Library/psbp-library.github.io.git
```
Comment (griggt) [inline .gitmodules:195]:
```suggestion
	url = https://github.com/PSBP-Library/psbp-library.github.io.git
```

File: .gitmodules [hunk 2]
```diff
@@ -193,3 +190,6 @@
 [submodule "community-build/community-projects/cask"]
 	path = community-build/community-projects/cask
 	url = https://github.com/dotty-staging/cask.git
+[submodule "community-build/community-projects/psbp"]
+	path = community-build/community-projects/psbp
+	url = https://github.com/PSBP-Library/psbp-library.github.io.git
```
Comment (OlivierBlanvillain) [inline .gitmodules:195]: I think we should keep pointing to our fork, but I didn't follow the latest community build changes...
wdyt @anatoliykmetyuk?

**Comment by OlivierBlanvillain** [comment]
> This is despite the repo being public. I can clone it locally no problem. Do you have an idea what could be going on?

That's weird, could it related to using https instead of git for the submodule?

**Comment by abgruszecki** [comment]
> > This is despite the repo being public. I can clone it locally no problem. Do you have an idea what could be going on?
> 
> That's weird, could it related to using https instead of git for the submodule?

Yep, that was the case. @griggt showed me how to fix it (thanks!).

File: .gitmodules (hunk 2 above)
Comment (anatoliykmetyuk) [reply .gitmodules:195]: Yes, that’s right. I created a fork: https://github.com/dotty-staging/psbp-library.github.io
And invited @PSBP-Library there. Please point to this repo.

**Review by anatoliykmetyuk — APPROVED** [review]

**Pull request merged**